Gemfile.lock

```diff
@@ -100,6 +100,9 @@ GIT
   specs:
     hyku_knapsack (0.0.1)
       rails (>= 5.2.0)
+      sentry-rails
```

Comment:
should it not be? We added sentry for hyku up knapsack and knapsack prime, but now that you question it I wonder if we should use bundler.d instead?

Reply:
TODO: rip it out of hyku and put it in bundler.d of the respective repo.

Reply:
This appears to have come about because we added sentry (et al) to the gem spec of knapsack; and when we bundled along came sentry.  This is definitely a surprise as we think about traditional directional dependencies; that is we think of Knapsack as laying on top of Hyku; but in reality it's a much more convoluted relationship.